src/StatusQ/Popups/Dialog/StatusHeaderActions.qml

```diff
@@ -0,0 +1,66 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQml.Models 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+Item {
+    id: root
+
+    property ObjectModel customButtons
+
+    property alias closeButton: closeButton
+    property alias infoButton: infoButton
+
+    height: implicitHeight
+    width: implicitWidth
+    implicitHeight: layout.implicitHeight
+    implicitWidth: layout.implicitWidth
+
+    RowLayout {
+        id: layout
+
+        anchors.fill: parent
+
+        Repeater {
+            model: root.customButtons
+            onItemAdded: {
+                item.Layout.fillHeight = true
+                item.Layout.preferredHeight = 32
+                item.Layout.preferredWidth = 32
+            }
+        }
+
+        StatusFlatRoundButton {
+            id: infoButton
+
+            Layout.fillHeight: true
+            Layout.preferredHeight: 32
+            Layout.preferredWidth: 32
+
+            visible: false
```

Comment:
Why create it when it's hidden by default?

Reply:
Cause it's more convenient to hide the info button by default, it's rarely used, see: https://www.figma.com/file/MtAO3a7HnEH5xjCDVNilS7/%F0%9F%8E%A8-Design-System-%E2%8E%9C-Desktop?node-id=993%3A5202

Reply:
No I meant, wrap it in a `Loader` :)

Reply:
I don't consider wrapping it into `Loader` would bring any value or performance gain. 
Also, it would break the consistency in the [interface](https://github.com/status-im/StatusQ/blob/fcc600fa4861b31cbbc9a647386289767b213c1d/src/StatusQ/Popups/Dialog/StatusHeaderActions.qml#L14-L15).

no laoder
```
actions.closeButton.visible: false
actions.infoButton.visible: true
actions.infoButton.onClicked: ...
```

vs

loader
```
actions.closeButton.visible: false
actions.infoButtonVisible: true
actions.infoButtonItem.onClicked: ... // this one wouldn't work if exposed through `loader.item`
```